mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -8,6 +8,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	FIELD field_22418 climbingPos Ljava/util/Optional;
 	FIELD field_23128 SOUL_SPEED_BOOST_ID Ljava/util/UUID;
 	FIELD field_27859 POWDER_SNOW_SLOW_ID Ljava/util/UUID;
+	FIELD field_30082 noInertia Z
```

Comment:
This seems a bit misleading- is this not more specifically related to having no friction?

Reply:
The way it's implemented doesn't seem like it really cares whether the entity is on the ground or not. Someone else is welcome to double-check that, though.

Reply:
There is still friction from air when off the ground

Reply:
So you think change it to `noFriction`? If you make a suggestion I'll happily accept.

Reply:
```suggestion
	FIELD field_30082 noDrag Z
```
This is drag. Drag is like multiplied onto velocity as a negative penalty.

Inertia is the tendency to keep the current velocity. You naming is exactly wrong because if there is no inertia, the entity should lose all velocity in the next tick.

Friction is a force that's applied when touching a surface. This is not friction either.

So `noDrag`. There is nothing resembling real-life friction or inertia in Minecraft!